src/main/database.cpp

```diff
@@ -272,6 +272,8 @@ void DatabaseInstance::Initialize(const char *database_path, DBConfig *user_conf
 		config.options.temporary_directory = string();
 	}
 
+	extension_api_v0 = make_uniq<duckdb_ext_api_v0>(CreateAPIv0());
+
```

Comment:
Just a note: `CreateAPIv0` can be tagged as constexpr starting in C++14, (not in C++11 due to more restrictive rules on assigments within constexpr) so I would assume most compilers to compile this away to just a bunch of memsets.